manifests/Microsoft/VisualStudioCode/1.46.1.yaml

```diff
@@ -0,0 +1,14 @@
+Id: Microsoft.VisualStudioCode
+Version: 1.46.1
+Name: Ladislau Szomoru
+Publisher: Microsoft Corporation
+Homepage: https://visualstudio.microsoft.com/downloads/
```

Comment:
This should be our homepage: https://github.com/microsoft/winget-pkgs/blob/f81c70010bc660c5b838f0d8dc3ef66889605bbb/manifests/Microsoft/VisualStudioCode/1.44.2.yaml#L5